src/cmd_line/commands/file.ts

```diff
@@ -83,31 +89,38 @@ export class FileCommand extends node.CommandBase {
       }
     } else {
       // Using a filename, open or create the file
-      this._arguments.name = <string>untildify(this._arguments.name);
+      this._arguments.name = this._arguments.name.replace('^file://', '');
```

Comment:
@jpoon `'file:///Users/user/test.js'.replace('^file://', '')` returns
"file:///Users/user/test.js"
Maybe it should be `.replace('^(file://)', '')`